librz/analysis/var_global.c

```diff
@@ -0,0 +1,432 @@
+// SPDX-FileCopyrightText: 2021 Basstorm <basstorm@nyist.edu.cn>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_analysis.h>
+#include <rz_list.h>
+
+/**
+ * \brief Create a new instance of global variable
+ * 
+ * \param name variable name
+ * \param addr variable address
+ * \param comment variable comment
+ * \return RzAnalysisVarGlobal *
+ */
+RZ_API RZ_OWN RzAnalysisVarGlobal *rz_analysis_var_global_new(RZ_NONNULL const char *name, ut64 addr, RZ_NULLABLE const char *comment) {
+	rz_return_val_if_fail(name && addr, NULL);
```

Comment:
```suggestion
	rz_return_val_if_fail(name, NULL);
```
Can't we have a global at 0 ?

Reply:
Yes, surely we can.